src/plugins/auto/src/executable_network.cpp

```diff
@@ -34,4 +34,11 @@ void ExecutableNetwork::SetExeNetworkForContext() {
 std::string ExecutableNetwork::GetLogTag() const noexcept {
     return _sContext->_LogTag;
 }
+
+std::shared_ptr<ngraph::Function> ExecutableNetwork::GetExecGraphInfo() {
+    // TODO: When there are more then 2 executable networks, such as MULTI:GPU,CPU
```

Comment:
I suppose in this case we can return an execution model for the graph with higher priority.

Reply:
Agree.